docs/dev-environments/ethereum/ganache/startup.sh

```diff
@@ -14,46 +14,38 @@
 # limitations under the License.
 
 
-check_and_exit()
+function error_exit()
 {
-    if [ $? -ne 0 ]
-    then
-        echo "Script failed."
-        exit 1
-    else
-        echo "Done."
-    fi
+    echo "[Error]: " ${*}
+    exit 1
 }
+
 echo ""
 echo "========================================================="
 echo "STEP 1 :: Create specific network for Ganache setup"
 echo "========================================================="
 
-docker network create -d bridge --subnet=172.18.0.0/16 ganache_local_net
-check_and_exit
+docker network create -d bridge --subnet=172.18.0.0/16 ganache_local_net \
+|| error_exit "Failed to create docker network for Ganache"
+echo "Done"
 
 echo ""
 echo "========================================================="
 echo "STEP 2 :: Start Ganache cli locally"
 echo "========================================================="
 
-docker run -d -p 8545:8545 --network=ganache_local_net --hostname=local-ganache --name=local-ganache trufflesuite/ganache-cli:v6.9.1
-check_and_exit
+docker run -d -p 8545:8545 --network=ganache_local_net --hostname=local-ganache \
+--name=local-ganache trufflesuite/ganache-cli:v6.9.1 || error_exit "Failed to start Ganache container"
+echo "Done"
```

Comment:
Can we create a docker-compose for ganache which can create a network and start the docker container

Reply:
It's only a single node setup which is why I haven't created a compose file for it.

Reply:
For consistency with besu it would be good if we can use a docker-compose file. If it's not too much of a work let's do it. If not, leave it for now.